utils/logger.js

```diff
@@ -1,9 +1,10 @@
-const httpErrorLogger = (err, message = 'Not message loaded') => console.log({
-	message,
-	errno: err.errno,
-	url: err.config.url
-});
-
+/* eslint-disable no-console */
 module.exports = {
-	httpErrorLogger
+	err: (message = 'Unknown Error', err) => console.error({
+		message: `[${new Date().toISOString()}]: ${message}`,
+		error: err.message,
+		stack: err.stack
+	}),
+	info: (message) => console.info(`[${new Date().toISOString()}]: ${message}`),
+	warn: (message) => console.warn(`[${new Date().toISOString()}]: ${message}`)
 };
```

Comment:
Better practice to move all of these to functions. So like 
```js
const err = (...) => {};
const info = (...) => {};
const warn = (...) => {}
module.exports = {
  err,
  info,
  warn
};
```

Reply:
very clever